### PR TITLE
Add `comment-on-draft-pr-indicator` feature

### PR DESCRIPTION
**Pull request by fregante**
Closes https://github.com/sindresorhus/refined-github/issues/3857
Based on #4520 

## Test

Any draft PR

### https://github.com/fregante/webext-detect-page/pull/2

- "Comment" button, repeatedly

### https://github.com/fregante/webext-detect-page/pull/2/files

- "Add single comment" button
- "Add review comment" button, except when responding to the same thread. See #4542 
- "Start a review" button

## Screenshots

Screen Shot 14

Screen Shot

### CODE REVIEW

**Comment by fregante** [comment]
@yakov116 you mentioned the need for `deduplicate`, but I don't see it here. Can you double-check?

**Review by yakov116 — CHANGES_REQUESTED** [review]
Does not stay after commenting. I _think_ just doing deduplicate false will work
Video_2021-07-06_133522

**Comment by darkred** [comment]
I just want to clarify one thing:
the above issue is the exact same I had with my PR:
the button wouldn't stay after updating without `onNewComments`
and setting `deduplicate: 'false'` didn't help.

**Comment by fregante** [comment]
The **"Close with comment | Comment"** part is a partial that is updated independently of new comments. I used a node removal listener to track its replacement

gif


> and setting `deduplicate: 'false'` didn't help.

`deduplicate` accepts selectors and `false`, not `'false'`

File: readme.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ Thanks for contributing! 🦋🙌
 - [](# "wait-for-attachments") [Wait for the attachments to finish uploading before allowing to post a comment.](https://user-images.githubusercontent.com/46634000/104294547-9b8b0c80-54bf-11eb-93e5-65ae158353b3.gif)
 - [](# "delete-review-comments-faster") [Adds a button to delete review comments in one click when editing them.](https://user-images.githubusercontent.com/46634000/115445792-9fdd6900-a216-11eb-9ba3-6dab4d2f9d32.png)
 - [](# "avoid-accidental-submissions") [Disables the <kbd>enter</kbd>-to-submit shortcut in some commit/PR/issue title fields to avoid accidental submissions. Use <kbd>ctrl</kbd><kbd>enter</kbd> instead.](https://user-images.githubusercontent.com/723651/115148453-5818df00-a068-11eb-92d4-51ac6e937b8b.gif)
+- [](# "comment-on-draft-pr-indicator") [Reminds you you’re commenting on a draft PR by mentioning it in the submit buttons.](https://user-images.githubusercontent.com/723651/123682081-39e31280-d853-11eb-9c86-97be400aca67.jpg)
```
Comment (fregante) [inline readme.md:205]:
```suggestion
- [](# "comment-on-draft-pr-indicator") [Reminds you you’re commenting on a draft PR by changing the submit button’s label.](https://user-images.githubusercontent.com/723651/123682081-39e31280-d853-11eb-9c86-97be400aca67.jpg)
```

File: source/helpers/on-replaced-element.ts [hunk 2]
```diff
@@ -9,18 +9,26 @@ Tracks the replacement of an element, identified via selector.
 @param callback The function to call after it's replaced
 */
 
-export default async function onReplacedElement(selector: string, callback: VoidCallback): Promise<void> {
+export default async function onReplacedElement(
+	selector: string,
```
Comment (yakov116) [inline source/helpers/on-replaced-element.ts:13]: Shouldn't this be changed to an object?  Isn't it a boolean trap?

Comment (fregante) [reply source/helpers/on-replaced-element.ts:13]: I'm not sure I follow

Comment (yakov116) [reply source/helpers/on-replaced-element.ts:13]: https://ariya.io/2011/08/hall-of-api-shame-boolean-trap

Comment (fregante) [reply source/helpers/on-replaced-element.ts:13]: Oh you meant to comment on the `callNow` line since there's no boolean here 😃 

Good point, changing

Comment (yakov116) [reply source/helpers/on-replaced-element.ts:13]: Sorry I was on mobile and selected the wrong line.

**Pull request merged**